config/settings.yml

```diff
@@ -132,10 +132,9 @@ mdot:
 
 # Settings for Decision Reviews
 decision_review:
-  prefill: true
-  url: https://sandbox-api.va.gov/services/appeals/v3/decision_review
-  api_key: abcd1234abcd1234abcd1234abcd1234abcd1234
```

Comment:
Is Settings.decision_review.api_key no longer needed? I still see it referenced in the base_request_headers method in DecisionReview::Configuration.

Reply:
@LindseySaari Whoops! Digging into this, I think my team has been putting settings in the wrong place.
https://github.com/department-of-veterans-affairs/vets-api/pull/4313